docs/source/extensions.rst

```diff
@@ -0,0 +1,61 @@
+.. _extensions-catalog:
+
+********************************
+Extensions Catalog and Proposals
+********************************
+
+NWB Extensions (NDX) Catalog
+============================
+
+The :ndx-catalog:`NDX Catalog website <>` contains published
+extensions created by the community to add support for a wide variety of data types.
+Before creating your own extension, we recommend that you explore the Catalog to
+make sure that someone has not already created an extension that suits your needs
+or that you could build off of.
+
+Policies on Community-driven Development and Integration of New Neuroscience Technologies with NWB
+
+- `Sharing Guidelines: requirements and strategy for sharing format extensions for NWB <https://docs.google.com/document/d/e/2PACX-1vRxbT-EEAyYbQL3P0TREpySJkMhV7ea2-aRO75_s4PhqzxnJa9p-s0SzVWrlkzEBaTw82bgzZBtxEuj/pub>`_
+- `Sharing Strategies: standard practices and strategies for sharing format extensions for NWB <https://docs.google.com/document/d/e/2PACX-1vSpLLPQV2XlfT-Qnpi_aqLPJzRjCko6Ur0U5COCEAQg5uLIN0h5vej5EPtsf6UNx1qiAIKXPiIveSWo/pub>`_
+- `Versioning Guidelines: requirements and strategy for versioning namespaces for the NWB core schema and extensions <https://docs.google.com/document/d/e/2PACX-1vSH72zNSUBToVcZDRI4gF7h15ImWRffvj-ju1oEbxggPrEFJd5L6GQc-fRiVmIi42U742tgjcRk65jv/pub>`_
+
+NWB Enhancement Proposals (NEPs)
+================================
+
+The NWB data standard is not static but evolves to adapt to changing needs from the neuroscience community, 
+clarify ambiguities, and fix bugs, among others. Users can propose smaller changes, such as improvements to
+documentation or addition of a new optional field to an existing data type, on the 
+:nwb-schema-src:`NWB Schema GitHub repository <>`. Users can propose larger changes, such as addition of 
+several new data types to support a new data modality, or a major restructuring of existing data types, 
+through NWB Enhancement Proposals (NEPs).
+
+NEPs are often collaborative and led by members of the community to address a significant need by the 
+community. Here is a list of NEPs that have been started:
+
+.. list-table:: NEPs
+   :widths: 15 45 25 15
+   :header-rows: 1
+
+   * - NEP
+     - Title
+     - Lead
+     - URLs
+   * - NEP001
+     - Events and TTL Data
+     - Ryan Ly
+     - `Google Doc <https://docs.google.com/document/d/1qcsjyFVX9oI_746RdMoDdmQPu940s0YtDjb1en1Xtdw/edit?usp=sharing>`_
+   * - NEP002
+     - Probe devices and channel mapping in extracellular ephys
+     - Alessio Buccino
+     - `Google Doc <https://docs.google.com/document/d/1q-haFEEHEgZpRoCzzQsuSWCKN4QfMsTzLnlptLaf-yw/edit?usp=sharing>`_
+
```

Comment:
I would recommend maintaining this list of NEP's in one place, preferably the place where NEPs are being managed, i.e., either on the repo or the NDX Catalog. It'll be hard enough to maintain this list in one place. I would just point to the list here.

Reply:
Yes, I suggest this is the only place we maintain this list.

Reply:
Should we have single folder in GoogleDrive where each NEP would get a subfolder such that we could simply embed the folder listing?

Reply:
I'm not sure if this is right place for this in the long run. I think something on nwb.org (once we have a new home) or on the NDX Catalog would be more appropriate.

Reply:
Interesting idea. Do you mean use an iframe here that points to a public Google Drive folder? Do you envision copying the google docs from the proposer's space to our folder or linking out to their google docs?

Reply:
I think either way is fine. In general it seems simpler to just have the documents (with one subfolder ber NEP) in the Drive directly, but linking to the documents seems perfectly fine, as long as there is public read access on the docs.

Reply:
> I think something on nwb.org (once we have a new home) or on the NDX Catalog would be more appropriate.

I think new nwb.org would be a good home. I think linking to the docs is easy and fine.